app/grandchallenge/workstation_configs/models.py

```diff
@@ -264,15 +264,45 @@ def save(self, *args, **kwargs):
 
 class WindowPreset(TitleSlugDescriptionModel):
     width = models.PositiveIntegerField(
-        validators=[MinValueValidator(limit_value=1)]
+        blank=True, null=True, validators=[MinValueValidator(limit_value=1)]
+    )
+    center = models.IntegerField(blank=True, null=True)
+
+    lower_percentile = models.PositiveSmallIntegerField(
+        blank=True, null=True, validators=[MaxValueValidator(limit_value=100)]
```

Comment:
Could you add a `MinValueValidator` to both the lower and upper percentiles, and test these?

Reply:
Thinking about it this is not necessary as it is implicitly included with the `PositiveSmallIntegerField`.